modules/combined/examples/periodic_strain/global_strain_pfm.i

```diff
@@ -0,0 +1,364 @@
+    tensor_values = '0 0 0 0 0 0.5'
+    tensor_name = shear1
+  [../]
+  [./shear2]
+    type = GenericConstantRankTwoTensor
+    tensor_values = '0 0 0 0 0 -0.5'
+    tensor_name = shear2
+  [../]
+  [./expand3]
+    type = GenericConstantRankTwoTensor
+    tensor_values = '1 1 0 0 0 0'
+    tensor_name = expand3
+  [../]
+
+  [./weight1]
+    type = DerivativeParsedMaterial
+    function = '0.3*c^2'
+    f_name = weight1
+    args = c
+  [../]
+  [./weight2]
+    type = DerivativeParsedMaterial
+    function = '0.3*(1-c)^2'
+    f_name = weight2
+    args = c
+  [../]
+  [./weight3]
+    type = DerivativeParsedMaterial
+    function = '4*(0.5-c)^2'
+    f_name = weight3
+    args = c
+  [../]
+
+  [./elasticity_tensor]
+    type = ComputeElasticityTensor
+    C_ijkl = '1 1'
+    fill_method = symmetric_isotropic
+  [../]
+  [./strain]
+    type = ComputeSmallStrain
+    global_strain = global_strain
+    eigenstrain_names = eigenstrain
+  [../]
+
+  [./eigenstrain]
+    type = CompositeEigenstrain
+    tensors = 'shear1  shear2  expand3'
+    weights = 'weight1 weight2 weight3'
+    args = c
+    eigenstrain_name = eigenstrain
+  [../]
+
+  [./global_strain]
+    type = ComputeGlobalStrain
+    scalar_global_strain = global_strain
+  [../]
+
+  [./stress]
+    type = ComputeLinearElasticStress
+  [../]
+
+  # chemical free energies
+  [./chemical_free_energy]
+    type = DerivativeParsedMaterial
+    f_name = Fc
+    function = '4*c^2*(1-c)^2'
+    args = 'c'
+    outputs = exodus
+    output_properties = Fc
+  [../]
+
+  # elastic free energies
+  [./elastic_free_energy]
+    type = ElasticEnergyMaterial
+    f_name = Fe
+    args = 'c'
+    outputs = exodus
+    output_properties = Fe
+  [../]
+
+  # free energy (chemical + elastic)
+  [./free_energy]
+    type = DerivativeSumMaterial
+    block = 0
+    f_name = F
+    sum_materials = 'Fc Fe'
+    args = 'c'
+  [../]
+[]
+
+[UserObjects]
+  [./appl_stress]
+    type = GlobalStrainUserObject
+    execute_on = 'Initial Timestep_begin Linear'
+  [../]
+[]
+
+[Postprocessors]
+  [./total_free_energy]
+    type = ElementIntegralVariablePostprocessor
+    execute_on = 'initial TIMESTEP_END'
+    variable = local_energy
+  [../]
+  [./total_solute]
+    type = ElementIntegralVariablePostprocessor
+    execute_on = 'initial TIMESTEP_END'
+    variable = c
+  [../]
+  [./min]
+    type = ElementExtremeValue
+    execute_on = 'initial TIMESTEP_END'
+    value_type = min
+    variable = c
+  [../]
+  [./max]
+    type = ElementExtremeValue
+    execute_on = 'initial TIMESTEP_END'
+    value_type = max
+    variable = c
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  scheme = bdf2
+  solve_type = 'PJFNK'
+
+  line_search = basic
+
+  petsc_options_iname = '-pc_type -sub_pc_type -pc_factor_shift_type -pc_factor_shift_amount'
+  petsc_options_value = ' asm      lu              NONZERO               1e-10'
```

Comment:
the `-pc_factor_shift_type -pc_factor_shift_amount` options are unnecessary. They are only required for the Lagrange multiplier approach in the mortar method.